presto-main/src/main/java/com/facebook/presto/execution/StageExecutionInfo.java

```diff
@@ -169,6 +171,9 @@ public static StageExecutionInfo create(
                     operatorToStats.compute(id, (k, v) -> v == null ? operatorStats : v.add(operatorStats));
                 }
             }
+            mergedRuntimeStats.mergeWith(taskStats.getRuntimeStats());
+            mergedRuntimeStats.addMetricValue("splitCountPerTask", taskStats.getTotalDrivers());
+            mergedRuntimeStats.addMetricValue("taskElapsedTimeNanos", taskStats.getElapsedTimeInNanos());
```

Comment:
Do we think it might be useful to make this string a `private static final` constant? No strong opinion given it is only used once here.

Reply:
Good point. Added a class for tracking all the RuntimeMetricName used in the core presto code base. This may be helpful to figure out what are the existing metrics and trace where they are populated. But for connectors, we allow it to use any names as we don't force it to update the core presto code base.